[2022-05-20 04:50:15,720] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: league_of_legends_dag.paths.path_items.Create_StagingSnowflake_Tables_items scheduled__2022-03-31T21:00:00+00:00 [queued]>
[2022-05-20 04:50:15,729] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: league_of_legends_dag.paths.path_items.Create_StagingSnowflake_Tables_items scheduled__2022-03-31T21:00:00+00:00 [queued]>
[2022-05-20 04:50:15,729] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-20 04:50:15,730] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-05-20 04:50:15,730] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-05-20 04:50:15,740] {taskinstance.py:1264} INFO - Executing <Task(SnowflakeOperator): paths.path_items.Create_StagingSnowflake_Tables_items> on 2022-03-31 21:00:00+00:00
[2022-05-20 04:50:15,745] {standard_task_runner.py:52} INFO - Started process 5178 to run task
[2022-05-20 04:50:15,748] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'league_of_legends_dag', 'paths.path_items.Create_StagingSnowflake_Tables_items', 'scheduled__2022-03-31T21:00:00+00:00', '--job-id', '671', '--raw', '--subdir', 'DAGS_FOLDER/dataDragon_dag.py', '--cfg-path', '/tmp/tmpe5jd3c7n', '--error-file', '/tmp/tmpwd3mn9tl']
[2022-05-20 04:50:15,748] {standard_task_runner.py:77} INFO - Job 671: Subtask paths.path_items.Create_StagingSnowflake_Tables_items
[2022-05-20 04:50:15,793] {logging_mixin.py:109} INFO - Running <TaskInstance: league_of_legends_dag.paths.path_items.Create_StagingSnowflake_Tables_items scheduled__2022-03-31T21:00:00+00:00 [running]> on host ca9a242fab95
[2022-05-20 04:50:15,840] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=league_of_legends_dag
AIRFLOW_CTX_TASK_ID=paths.path_items.Create_StagingSnowflake_Tables_items
AIRFLOW_CTX_EXECUTION_DATE=2022-03-31T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-31T21:00:00+00:00
[2022-05-20 04:50:15,841] {snowflake.py:118} INFO - Executing: CREATE OR REPLACE TABLE STAGING_DATADRAGON_ITEMS (json_data variant);
[2022-05-20 04:50:15,851] {connection.py:257} INFO - Snowflake Connector for Python Version: 2.5.1, Python Version: 3.8.12, Platform: Linux-5.10.102.1-microsoft-standard-WSL2-x86_64-with-glibc2.2.5
[2022-05-20 04:50:15,865] {connection.py:868} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2022-05-20 04:50:15,865] {connection.py:886} INFO - Setting use_openssl_only mode to False
[2022-05-20 04:50:16,645] {cursor.py:637} INFO - query: [ALTER SESSION SET autocommit=True]
[2022-05-20 04:50:16,791] {cursor.py:661} INFO - query execution done
[2022-05-20 04:50:16,792] {snowflake.py:324} INFO - Running statement: CREATE OR REPLACE TABLE STAGING_DATADRAGON_ITEMS (json_data variant);, parameters: None
[2022-05-20 04:50:16,792] {cursor.py:637} INFO - query: [CREATE OR REPLACE TABLE STAGING_DATADRAGON_ITEMS (json_data variant);]
[2022-05-20 04:50:16,988] {cursor.py:661} INFO - query execution done
[2022-05-20 04:50:16,989] {snowflake.py:334} INFO - Statement execution info - {'status': 'Table STAGING_DATADRAGON_ITEMS successfully created.'}
[2022-05-20 04:50:16,989] {json_result.py:127} INFO - fetching data done
[2022-05-20 04:50:16,989] {snowflake.py:338} INFO - Rows affected: 1
[2022-05-20 04:50:16,990] {snowflake.py:339} INFO - Snowflake query id: 01a462a2-0000-f1b3-0000-d969000378a6
[2022-05-20 04:50:16,990] {connection.py:499} INFO - closed
[2022-05-20 04:50:17,046] {connection.py:502} INFO - No async queries seem to be running, deleting session
[2022-05-20 04:50:17,165] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=league_of_legends_dag, task_id=paths.path_items.Create_StagingSnowflake_Tables_items, execution_date=20220331T210000, start_date=20220520T045015, end_date=20220520T045017
[2022-05-20 04:50:17,205] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-20 04:50:17,245] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
